### DISCUSSION TITLE
【原神】4.0大更新来临

### DISCUSSION THREAD

**Discussion by zzz14121** [share121/inter-knot#1301] [其他]
![0a7ab50f95f4b4bf2f1a727c018c5e33401742377](https://github.com/user-attachments/assets/d1821e25-ddcf-4239-9873-2be74065ac74)
奉上海露港照片一张

- *（仅供参考，可做壁纸）

**Comment by EVidosilence** [comment]
![9bf7266c17be08d8592cf2103226c8f5](https://github.com/user-attachments/assets/277d84dd-3cb7-4402-b9d2-ad060f58daf0)

**Comment by Jyao6011** [comment]
怎么哪里都有原😡
